book/website/communication/open/education.md

```diff
@@ -0,0 +1,82 @@
+(cm-open-education)=
+# Open Education 
+
+(cm-open-education-what)=
+## What is Open Education
+Open Education is a collective term that encompasses educational resources, tools and practices that can be freely used in the digital environment without legal, financial or technical barriers. 
+Open Education can also include [Open Pedagogy](http://openpedagogy.org/open-pedagogy/) which also focuses on the conversation with each other to develop educational practices and structures. 
```

Comment:
Would re-phrase "on the conversation with each other to develop educational practices and structures." to "which also focuses on the process of collaboratively developing educational practices and structures"

Reply:
Changed, thanks!